test/hbbft_relcast_distributed_SUITE.erl

```diff
@@ -136,3 +136,234 @@ simple_test(Config) ->
 
     [ unlink(W) || {_, {ok, W}} <- Workers ],
     ok.
+
```

Comment:
are these just dupes of the other stuff but run through the relcast worker?

Reply:
yes